app/main.py

```diff
@@ -2,0 +1,12 @@
+def calculate_team_total_rating(team: int) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elves: int) -> None:
+    for elf in elves:
+        elf.play_elf_song()
+
+
+def feast_of_the_dwarves(dwarves: int) -> None:
```

Comment:
The parameter `dwarves` should be a list of `Dwarf` objects, not an integer. Update the type annotation to `List[Dwarf]`.